sda-commons-server-morphia-example/src/test/java/org/sdase/commons/server/morphia/example/MorphiaApplicationIT.java

```diff
@@ -29,8 +29,11 @@
           randomPorts(),
           // provide a lambda to only read the value after the mongodb connection parameters are
           // available
-          config("mongo.hosts", MONGODB::getHost),
-          config("mongo.database", MongoDbRule.Builder.DEFAULT_DATABASE));
+          config("mongo.hosts", MONGODB::getHosts),
+          config("mongo.database", MONGODB::getDatabase),
+          config("mongo.username", MONGODB::getUsername),
+          config("mongo.password", MONGODB::getPassword),
+          config("mongo.options", MONGODB::getOptions));
```

Comment:
Not related to your changes: We could provide a getter "MONGODB::getConfig" to set the "mongo" property in the config with only one statement.

Reply:
That was my first idea as well, but mongo-testing intentionally has no dependency to server-morphia where the config class resides. A new module for a single class seems too much for this PR in my opinion.